test/compose/test-compose

```diff
@@ -240,7 +241,10 @@ function podman() {
         --runroot $WORKDIR/runroot \
         --network-config-dir $WORKDIR/cni \
         "$@")
+    rc=$?
+
     echo -n "$output" >>$WORKDIR/output.log
+    return $rc
```

Comment:
`podman` wrapper did not pass through the return code which broke the compose result checks.